istio/tests/cases/configuration.py

```diff
@@ -0,0 +1,72 @@
+import subprocess
+import grequests
+import performance
+import time
+VERSION='0.6.0'
+rule_apply_time=5
+# istio_path="../../release/samples/bookinfo/kube/"
+istio_path="nginmesh-0.6.0/samples/bookinfo/kube/"
```

Comment:
this is not istio path. it is nginmesh path

Reply:
refer to local source path not release

Reply:
ok